es/README.md

```diff
@@ -1,36 +1,70 @@
-## Install, configure and start Elasticsearch
+# Install, configure and start Elasticsearch
 
-### Manual installation
+## Manual installation
 
-Download Elasticsearch (at least 7.6.2 for Geonetwork 3.8.x) from https://www.elastic.co/fr/downloads/elasticsearch
-and copy it to the ES module. eg. es/elasticsearch-7.6.2
+1. Download Elasticsearch (at least 7.6.2 for Geonetwork 4.0.x) from https://www.elastic.co/downloads/elasticsearch
```

Comment:
The ES version used currently is `7.11.1`, please check the different places using `7.6.2` to update the PR.

Reply:
I am just improving the docs here, not updating the version number.

- How would you like manage the version number used? Updating it in a bunch of documentation pages is not the easiest to maintain.
- We should stay with the open source release of ElasticSearch, the newer version is no longer open source

Reply:
To be clear a request to move past ES 7.10 should not be supported by my employer (as an open source company we do not wish to expose our customers to requiring a non open source component out of the box).

Optional closed-source technology like using an oracle database is fine, but only as a customer choice.